docs/distributions/ubuntu/installation.md

```diff
@@ -0,0 +1,51 @@
+# Download the latest safe release
+
+For mbp-ubuntu, we recommend using the 5.7.19 kernel because 5.8 and newer kernels can have problems.
```

Comment:
I'd stick closer to the phrasing from here https://github.com/marcosfad/mbp-ubuntu/pull/26#discussion_r550875065